### ISSUE TITLE
wxFormBuilder

### ISSUE THREAD

**Issue by GWRon** [HenriVi/Editor#2]
I do not know if you are aware of wxFormBuilder - with this tool you can "visually" design the form of dialogue windows, app windows ...

and the best thing is? checkout "wx.mod/tools"!
There is a wxCodeGen - which takes a wxFormBuilder-file and generates a form.bmx (or however you name it) base file setting up all the widgets as you did in wxFormBuilder.

In your main app file you just import this form.bmx then, and override event-listeners you defined within wxFormBuilder (so onClick-listeners and so on).

**Comment by HenriVi** [comment]
Yes, wxFormBuilder is a great program to quickly see what wxWidgets is capable of. Great learning tool also.

With this particular project I prefer not to use it, to allow myself greater control over all aspects, and to keep the source as clean as possible.

The next main elements ahead in my mind are (but not necessary in this order):

- Get lexer working fool proof
- Add basic autocomplite, because it relates to lexer.
- Add Options menu to allow control over styling / theme
- Actually open / save files
- Add bmk-options to actually run something + output console
- Create explorer to browse code
- Add basic intellisense
- etc.

Most important thing is to keep everything sufficiantly basic and make sure all quirks are ironed out before adding any advanced stuff.

-Henri

**Comment by franciscocmo4ever** [comment]
Olá você sabe como posso fazer uma doação ao projeto wxFormBuilder????
Hello, do you know how I can make a donation to the wxFormBuilder project????
